index.html

```diff
@@ -87,6 +91,8 @@
             <br>
         </div>
         <br>
+        <div class="plyr"><audio></audio></div>
```

Comment:
Does the `<audio>` tag have any effect here? I didn't notice any difference when I was playing with it locally